translations/monero-core_it.ts

```diff
@@ -784,102 +784,102 @@
     <message>
         <location filename="../pages/Receive.qml" line="138"/>
         <source>With more Monero</source>
-        <translation type="unfinished"></translation>
+        <translation>Con più Moneroj</translation>
     </message>
     <message>
         <location filename="../pages/Receive.qml" line="139"/>
         <source>With not enough Monero</source>
-        <translation type="unfinished"></translation>
+        <translation>Con una quantità insufficiente di Moneroj</translation>
     </message>
     <message>
         <location filename="../pages/Receive.qml" line="141"/>
         <source>Expected</source>
-        <translation type="unfinished"></translation>
+        <translation>Atteso</translation>
```

Comment:
Expected is not "Atteso" but "Previsto", imho

Reply:
Expected can be translated with "atteso" and "previsto" depending on the context. (e.g. "expected value" is "valore atteso"). Since I cannot figure out the context and you say it is better "previsto" (and erciccione makes thumbs up), let's change! :-).
Agreed.